server/server.js

```diff
@@ -74,6 +76,7 @@ app.get('/products/:product_id/related', (req, res) => {
     .catch(err => res.end(JSON.stringify(err)));
 });
 
+// qqq delete when better API route for current product is completed
```

Comment:
qqq is a new one to me; Is that an industry standard, or a personal searchable term?
I'm used to seeing TODO

Reply:
I can't remember where I first read this (maybe a Tim Ferris book, but I'd rather credit anyone else), but it's very easy to search for and multiple q's never show up. Doubt it's a standard, hahaha